CHANGELOG.md

```diff
@@ -32,9 +32,10 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On Wayland, fix polling during consecutive `EventLoop::run_return` invocations.
 - On Windows, fix race issue creating fullscreen windows with `WindowBuilder::with_fullscreen`
 - On Android, `virtual_keycode` for `KeyboardInput` events is now filled in where a suitable match is found.
-- **Breaking:** Added new `WindowEvent::IME` supported on macOS, X11, and Wayland.
-- Added `Window::set_ime_allowed` supported on macOS, X11, and Wayland.
-- **Breaking:** IME input on macOS, X11, and Wayland won't be received unless it's explicitly allowed via `Window::set_ime_allowed` and new `WindowEvent::IME` events are handled.
+- **Breaking:** Added new `WindowEvent::IME` supported on macOS, Windows, X11, and Wayland.
+- Added `Window::set_ime_allowed` supported on macOS, Windows, X11, and Wayland.
+- **Breaking:** IME input on macOS, Windows, X11, and Wayland won't be received unless it's explicitly allowed via `Window::set_ime_allowed` and new `WindowEvent::IME` events are handled.
```

Comment:
Could you rename that `macOS, Windows, X11` etc stuff to `desktop platforms`? So it'll be `supported on desktop platforms.`

Reply:
done